services/datamanager/builtin/builtin.go

```diff
@@ -433,6 +437,28 @@ func (svc *builtIn) Reconfigure(
 				// do not have the resource right now
 				continue
 			}
+
+			captureFrequencyKey := fmt.Sprintf("%s-capture-frequency", resConf.Name.String())
+			_, ok := svc.statusChangedMap[captureFrequencyKey]
+
+			// only log if it's not set AND has not been logged before or if it's been reset
+			// otherwise we'll be logging way too much
+			if !ok && resConf.CaptureFrequencyHz == 0 {
+				svc.logger.Infof("capture frequency for component %s is not set and will not sync", resConf.Name)
+			} else if ok && resConf.CaptureFrequencyHz != svc.statusChangedMap[captureFrequencyKey] {
+				syncVal := ""
+				if resConf.CaptureFrequencyHz == 0 {
+					syncVal = "will not"
+				} else {
+					syncVal = "will"
+				}
+				svc.logger.Infof("capture frequency has been set to %f/s and %s sync", resConf.CaptureFrequencyHz, syncVal)
```

Comment:
This is a cool and useful idea to start logging the capture frequency in the RDK logs whenever it's toggled, rather just in the zero case! I think we'd just want to include the whole resourceMethodMetadata here, in some easy-to-read print representation.